src/Main.java

```diff
@@ -1,5 +1,44 @@
 public class Main {
     public static void main(String[] args) {
-        System.out.println("Hello world!");
+    // Задача 1
+    int [] loop = {55_300, 91_541, 68_546, 84_910, 78_053};
+    int sum = 0;
+    for (int i: loop) {
+        sum += i;
     }
-}
\ No newline at end of file
+
+    System.out.println("Сумма трат за месяц составила " + sum + " рублей.");
+    // Задача 2
+    int min = loop[0];
+    int max = loop[1];
```

Comment:
Тоже надо нулевую, вдруг самы первы элемент массива и есть самый большой
```suggestion
    int max = loop[0];
```